sql/updates/characters/2015_08_26_00_characters_335.sql

```diff
@@ -0,0 +1,16 @@
+-- Keep only the highest guid PvE or PvP (not bones) corpse per player guid
+DELETE c FROM `corpse` c LEFT JOIN
+(
+    SELECT MAX(`corpseGuid`) AS id
```

Comment:
Yeah I considered this might happen. But also saw someone else that used sql that would detect this in a pr shot down... Only a problem until next tdb anyway.